src/pre_defined_classes/simple_programs_taking_numeric_user_input/medium/q6/Intelligence.java

```diff
@@ -0,0 +1,31 @@
+package pre_defined_classes.simple_programs_taking_numeric_user_input.medium.q6;
+/*
+
```

Comment:
would this be a better way of wording the question?
```
A computer science teacher would like to ask the students about the intelligence of computers versus humans.
Make a program that asks the user the percentage of how many time they think computers are smarter than humans
assuming users always give a valid input (in this question: integer from 1 to 1000, inclusive).
```
```
Scanner input = new Scanner( System.in );
        Random gen = new Random();
        System.out.println( "Are computers smarter than humans? (0 for no, 1 for yes): " );
        System.out.println("How many time do you think computers are smarter than humans? (1 - 1000 , enter 1 for 1%)");
        int answer = input.nextInt();
        int pct = gen.nextInt( 1000 );
        System.out.println("You have entered " + pct+ " percent" );
```

Reply:
This is better, but I would still get the user to input 0 or 1 for yes or no, and then only have the random number go from 1 to 100 (because it is a percent). For example: 
```
public static void main(String[] args){
     System.out.println("Do you think computers are smarted than humans (0 for no, 1 for yes)?")
     Scanner input = new Scanner(System.in);
     int answer = intput.nextInt();
     Random gen = new Random();
     int correct = gen.nextInt(101);
     System.out.println(answer + " is " + correct + "% correct.")
}
```